tests/src/fixtures/parsers/components/target.yml

```diff
@@ -26,7 +26,6 @@
     getAcceptedDelay : "10/07/2007"
     getAcceptedDelayFormat : "dd/mm/YYYY"
     getTranslatedFile : "File"
-    getRemark : "Remark"
```

Comment:
Why has this been changed?

Reply:
The key is duplicated in the array and Phpunit is strict about it by default.